### PR TITLE
Add more information and checkbox to register descriptor step

### LINKED ISSUE #297: Installer: enhance register descriptor step
We should really make it clear they need to click on the signing devices tabs. We could also have a checkbox here (like in descriptor backup).

### PR DESCRIPTION
**Pull request by edouardparis**
2023-02-04T10:04:29,401091329+01:00
close #297

### CODE REVIEW

File: gui/src/installer/prompt.rs [hunk 1]
```diff
@@ -6,3 +6,4 @@ pub const DEFINE_DESCRIPTOR_SEQUENCE_TOOLTIP: &str =
     "Number of blocks after a coin is received \nfor which the recovery path is not available";
 pub const DEFINE_DESCRIPTOR_FINGERPRINT_TOOLTIP: &str =
     "The alias is applied on all the keys derived from the same seed";
+pub const REGISTER_DESCRIPTOR_HELP: &str = "In order for an external device to display correct information about the future transactions, the descriptor must be registered on it. Beware that this registration does not replace backup.";
```
Comment (darosior) [inline gui/src/installer/prompt.rs:9]:
```suggestion
pub const REGISTER_DESCRIPTOR_HELP: &str = "To be used with the wallet, a device needs the descriptor. Registration on a device is not a substitute for backing up the descriptor.";
```

File: gui/src/installer/view.rs [hunk 2]
```diff
@@ -640,7 +643,7 @@ pub fn register_descriptor<'a>(
                             .align_items(Alignment::Center)
                             .push(
                                 Container::new(
-                                    text(format!("{} hardware wallets connected", hws.len()))
+                                    text("Select hardware wallet to register descriptor on it:")
```
Comment (darosior) [inline gui/src/installer/view.rs:646]:
```suggestion
                                    text("Select hardware wallet to register descriptor on:")
```

File: gui/src/installer/view.rs [hunk 3]
```diff
@@ -668,12 +671,17 @@ pub fn register_descriptor<'a>(
                     )
                     .width(Length::Fill),
             )
-            .push(if processing {
-                button::primary(None, "Next").width(Length::Units(200))
-            } else {
+            .push(Checkbox::new(
+                "I have registered the descriptor on my devices",
```
Comment (darosior) [inline gui/src/installer/view.rs:675]:
```suggestion
                "I have registered the descriptor on my device(s)",
```

**Review by darosior — COMMENTED** [review]
Great! It's much clearer now.

**Comment by edouardparis** [comment]
fixed comments and rebased

**Review by darosior — COMMENTED** [review]
utACK 697fb16db976d0bc643381b10342f2df650a4386 -- text looks good to me.

**Pull request merged**